### DISCUSSION TITLE
No "select all" option in Edit menu or context menu in text editor

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk#55] [General]
This forces to use Ctrl+A...

**Comment by m417z** [comment]
Good point, as well as actions such as copy and paste. Added in Windhawk version 1.3 which was just released.

**Comment by Anixx** [comment]
Still, there is no "Select all" in context menu, only in "Selection" drop-down.

**Reply by m417z** [reply]
I thought that you were talking about menus in the app, not the editor, like this:

![image](https://user-images.githubusercontent.com/4129781/229270980-4078bdd2-efc2-46be-a3f5-29707421ff43.png)

The editor is based on VSCode, and they didn't add "Select all" for some reason. It was requested at some point but was rejected:
https://github.com/microsoft/vscode/issues/163674

**Comment by Anixx** [comment]
This is still missing from the context menu, even though it is now available in the pull-down menu. Having it in the context menu would be more handy.
